### ISSUE TITLE
Change Tornjak Frontend to use Nextjs

### ISSUE THREAD

**Issue by marcofranssen** [spiffe/tornjak#151]
To benefit from a more robust React Setup it would be great if Tornjak Frontend could be refactored to use [Next.js](https://nextjs.org/).

Using Next.js we can leverage Server Side Rendering as well Static Pages. This will give us an easier to maintain codebase, as well severe performance gains.

Refactors should be fairly straight forward to start using nextjs.

https://nextjs.org/docs/migrating/from-create-react-app

**Comment by marcofranssen** [comment]
I don't see Nextjs being used in the implementation on the v1.4 branch. I only see the dependency, but in essence the implementation is still a create-react-app. Can we reopen this issue> I would love to contribute and make this happen. Will require a lot of reorganizing the files and changing a bunch of components though.

**Comment by mamy-CS** [comment]
Yes @marcofranssen, I was going to open another issue specifically to re-organize the components. We just didn't have enough bandwidth to do it now. That will be great actually if you would like to contribute to it we can take care of it now. Thanks.

**Comment by mamy-CS** [comment]
Reopening issue, to take care of reorganizing the frontend components here.
